core/src/autogluon/core/trainer/abstract_trainer.py

```diff
@@ -871,6 +871,54 @@ def get_model_pred_proba_dict(self,
         else:
             return model_pred_proba_dict
 
+    def get_model_pred_dict(self,
+                            X: pd.DataFrame,
+                            models: List[str],
+                            record_pred_time: bool = False,
+                            **kwargs):
```

Comment:
add return type hint

Reply:
will wait for future PR, the return type is not trivial as it can be a tuple